### DISCUSSION TITLE
Is This A BUG?!  Unable to read shares database. Please rescan your shares. Error: tuple index out of range  ?!

### DISCUSSION THREAD

**Discussion by DeathStalker77** [nicotine-plus/nicotine-plus#2685] [Q&A] [answered]
I just got this message trying to start the program 0 v3.3.0 B5.  It is also hanging for a LONG time.  I will try forcing a close and update to the current nightly build, then update this.
--- DS

*Update:  Ok, it doesn't appear to give that error with the Build from the 9th (but there is still a MAJOR performance issue)

**Comment by mathiascode** [comment, marked as answer]
Try the latest build here: https://nicotine-plus.org/doc/TESTING.html#windows

Rescan your shares and let it finish completely. If the error still occurs after the rescan finished, let me know, and I'll look into it.
